media/js/base/mozilla-traffic-cop.js

```diff
@@ -72,6 +72,11 @@ Mozilla.TrafficCop.noVariationCookieValue = 'novariation';
 Mozilla.TrafficCop.prototype.init = function() {
     var redirectUrl;
 
+    // respect the DNT
+    if (window._dntEnabled() === true) {
```

Comment:
For an extra bit of resilience would it be worth adding check to make sure `_dntEnabled` is defined? You could also omit the `=== true` part.